From bc8e83fcc6132c63163a2b5b7f32d6b8c8575ad2 Mon Sep 17 00:00:00 2001
From: Matt Heon <mheon@redhat.com>
Date: Mon, 30 Oct 2023 10:37:37 -0400
Subject: [PATCH] Mask /sys/devices/virtual/powercap by default

This mimics #5111 but without the vendor update, which is very
large on this branch.

[NO NEW TESTS NEEDED] This cannot be tested in CI as cloud
providers already don't provide these interfaces in their VMs.

Signed-off-by: Matt Heon <mheon@redhat.com>
---
 run_linux.go | 1 +
 1 file changed, 1 insertion(+)

diff --git a/run_linux.go b/run_linux.go
index e3c12196f1..61b498b2b2 100644
--- a/run_linux.go
+++ b/run_linux.go
@@ -992,6 +992,7 @@ func setupMaskedPaths(g *generate.Generator) {
 		"/sys/firmware",
 		"/sys/fs/selinux",
 		"/sys/dev",
+		"/sys/devices/virtual/powercap",
 	} {
 		g.AddLinuxMaskedPaths(mp)
 	}
